packages/react-scripts/config/webpack.config.js

```diff
@@ -60,7 +60,7 @@ const sassModuleRegex = /\.module\.(scss|sass)$/;
 
 // This is the production and development configuration.
 // It is focused on developer experience, fast rebuilds, and a minimal bundle.
-module.exports = function(webpackEnv) {
+module.exports = function (webpackEnv) {
```

Comment:
We try our best not to edit this file with our prettier config, that way in the future it makes merging upstream changes easier on us. Could you undo the whitespace/prettier changes for this file?

Reply:
sure thing. been only working inside zion code lately, so i forgot to turn it off, it was on auto